### ISSUE TITLE
elix-carousel: Documentation of currentindexchange event is wrong

### ISSUE THREAD

**Issue by nerdbeere** [elix/elix#171]
According to the [elix-carousel docs](https://component.kitchen/elix/Carousel) there's an event `currentindexchanged` that triggers when the currentIndex property changes. Either the documentation or the [event name](https://github.com/elix/elix/blob/main/src/base/CursorAPIMixin.js#L169-L171) is incorrect.

**Steps that reproduce the issue**
1. Create a page with `<elix-carousel>`.
2. Add event listener to `currentindexchanged`
3. Click arrow to move to the next slide
4. Event is not emitted

**Comment by JanMiksovsky** [comment]
Many thanks for spotting this. We had changed the event names a long time ago, but until recently had left the deprecated names like `currentindexchanged` intact to allow people time to transition away from them.

We recently removed those deprecated event names, and did our best to remove them from the docs, but we missed a few.

I've merged this, although for some reason this morning I'm running into an issue updating the public docs site. Once I resolve that, the `currentindexchange` event will appear in the documentation.
